builtin/game/chat.lua

```diff
@@ -1068,7 +1038,7 @@ core.register_chatcommand("last-login", {
 			param = name
 		end
 		local pauth = core.get_auth_handler().get_auth(param)
-		if pauth and pauth.last_login and pauth.last_login ~= -1 then
```

Comment:
Old player auth will cause a value of `-1` due to lack of the `last_login` value in the records. Hence the login date will now be shown incorrectly.